pkg/exchange/bybit/stream.go

```diff
@@ -420,3 +436,28 @@ func (s *Stream) handleTradeEvent(events []TradeEvent) {
 		s.StandardStream.EmitTradeUpdate(*gTrade)
 	}
 }
+
+// symbolSplitter attempts to divide a trading pair into its base and quote currency. This is particularly useful for
+// calculating the fee currency when we are unable to retrieve the fee rate from the poller.
+//
+// Bybit represents currencies with uppercase in API responses.
+var symbolSplitter = regexp.MustCompile(`(\w+)(USDT|USDC|USD)$`)
```

Comment:
not using markets?

Reply:
This scenario represents the worst-case situation where we're unable to fetch fee rates due to the Poller's failure to function properly. 🥲